deploy/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.opennms</groupId>
+    <artifactId>opennms</artifactId>
+    <version>31.0.3-SNAPSHOT</version>
```

Comment:
I suspect, the version will be updated before delivery of the changes.

Reply:
it's being merged to `foundation-2023` so it will get changed just like everything else does, but yeah, I'll have to hand-fix it for the first merge forward to m2023 and release-31.x and such